src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -117,6 +153,42 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
         }
     }
 
+    /**
+     * Update the current camera state depending on the inputs that have been used this frame.
+     * This is a dynamically created lambda to avoid the performance penalty of looping for inputs in the render loop.
+     */
+    public checkInputs(): void {
+        if (this.wheelPrecisionEaseInRatio) {

```

Comment:
maybe just return direct (to remove a block:
if (!this.wheelPrecisionEaseInRatio) {
return;
}

Reply:
done.